lib/commands/recordscreen.js

```diff
@@ -49,8 +49,11 @@ class ScreenRecorder {
 
     const args = [
       '-f', 'mjpeg',
-      '-i', `http://localhost:${localPort}`,
     ];
+    if (this.opts.videoFps && this.opts.videoType === 'libx264') {
```

Comment:
should we apply it only for libx264 codec or always in case the codec is not equal to `DEFAULT_VCODEC`?

Reply:
perhaps if `parseInt(this.opts.videoFps, 10) > 0`

Reply:
Due to my investigations we need this option only for libx264, mjpeg works correct without this option. Not sure about behavior for other codecs with -r option, that's why I did only the check for libx264.

Reply:
About `parseInt,` we have this check before starting screen recording, lines 248-250.
`if (videoFps) {
    const fps = parseInt(videoFps, 10);
    if (isNaN(fps)) {
      throw new Error(`videoFps value should be a valid number in range 1..60. ` +
        `'${videoFps}' is given instead`);
    }`

If you think we need additional check here, I'll add it.

Reply:
yep, this makes sense. 

It would be useful to add a comment there containing the above explanation about the codec type (or just add a link to the Appium issue)